### PR TITLE
fix(tests) : updated few tests to wait for ready state

### PR DESCRIPTION
**Pull request by Raunak1203**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
This PR adds ready states for functions to avoid random failures and wait for the expected condition.
- [ ] What issue/task does this PR references?

- [x] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

File: src/tests/ui/planner/workitem-list.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { BaseElement } from './../base.element';
-import { ElementFinder, by } from 'protractor';
+import { ElementFinder, by, browser } from 'protractor';
```
Comment (jarifibrahim) [inline src/tests/ui/planner/workitem-list.ts]: Unnecessary import

Comment (Raunak1203) [reply src/tests/ui/planner/workitem-list.ts]: @jarifibrahim It is required https://github.com/Raunak1203/fabric8-planner/blob/30bcae1770e784035c3be0a14b934318d8fc3060/src/tests/ui/planner/workitem-list.ts#L74

Comment (jarifibrahim) [reply src/tests/ui/planner/workitem-list.ts]: okay

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Pull request merged**